config/routes.rb

```diff
@@ -56,11 +73,14 @@
 
   # Singleton routes for admin panel
   get '/administrator', to: 'administrator#index'
+
   get '/administrator/change_admin', to: 'administrator#changeAdmin'
   get '/administrator/change_editor', to: 'administrator#changeEditor'
   get '/administrator/change_basic', to: 'administrator#changeBasic'
-
-
+  get '/sort', to: 'administrator#sort'
```

Comment:
These routes did get used except for sort. The application crashes at the admin index page when I remove these routes.